config/traject/pdc_describe_indexing_config.rb

```diff
@@ -235,31 +235,6 @@
   end
 end
 
-# ==================
-# Store files metadata as a single JSON string so that we can display detailed information for each of them.
-#
-# TODO: Note that this information is duplicated with what we save in `pdc_describe_json_ss`. For large
-# datasets (e.g. those with 60K files) this is less than ideal. We should look into optimizing
-# this when we take care of https://github.com/pulibrary/pdc_discovery/issues/738
-to_field 'files_ss' do |record, accumulator, _context|
```

Comment:
This is the main change: we are not indexing this field anymore since the same data is available in `pdc_describe_json_ss`